cartography/intel/aws/rds.py

```diff
@@ -200,8 +200,8 @@ def _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag):
     Attach read replicas to their source instances
     """
     attach_replica_to_source = """
-    MATCH (replica:RDSInstance{id:{ReplicaArn}}),
-    (source:RDSInstance{db_instance_identifier:{SourceInstanceIdentifier}})
+    MATCH (replica:RDSInstance{id:{ReplicaArn}})
+    MERGE (source:RDSInstance{id:{SourceInstanceIdentifier}})
```

Comment:
Oh, sorry my comment was maybe a little vague - I wasn't commenting on _cartography_'s naming convention, but on AWS: they have a "bug" where the valu inside `ReadReplicaSourceDBInstanceIdentifier` is not, in fact, a `DBInstanceIdentifier` but rather an `ARN`. See attached image for an example:
![Screen Shot 2019-04-08 at 3 42 45 PM](https://user-images.githubusercontent.com/5350399/55761774-55189c00-5a15-11e9-9362-981d77caecd3.png)

I'll change some variable names to better show what I mean, as well (since it's definitely unclear the way I have it written).

Reply:
That is really weird - for us `ReadReplicaSourceDBInstanceIdentifier` is always a DB instance identifier and not an ARN .

Is it possible that your example's DB instance identifier is set to an "ARN-like" string?

Reply:
Ohhh, interesting. All the `DBInstanceIdentifiers` look well-formed (not `ARN`-like strings).

Our setup doesn't happen to have intra-region read-replicas at the moment - maybe having it be cross-region causes it to include the ARN? That seems especially insidious. I'll try to investigate when I can.

Reply:
Update:

in-region read-replicas have only the `DBInstanceIdentifier` while out-of-region has the entire `ARN`:
![Screen Shot 2019-04-10 at 10 32 53 AM](https://user-images.githubusercontent.com/5350399/55900577-5b755800-5b7c-11e9-838c-98d1e23b7e23.png)

Thankfully, we cannot create ambiguous `DBInstanceIdentifiers` that might suspiciously look like `ARNs`, so at least it's clear what we're looking at.
![Screen Shot 2019-04-10 at 10 33 12 AM](https://user-images.githubusercontent.com/5350399/55900586-5fa17580-5b7c-11e9-9afc-a1e9ceb05f89.png)